cmd/openshift-install/log-config.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"os"
+	"path"
+
+	"github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v2"
+)
+
+type LogConfig struct {
+	// Fields contains user-provided fields to be added to all log entries.
+	// +optional
+	Fields logrus.Fields `json:"fields,omitempty"`
+
+	// Level sets the level of logging to standard out.
+	// Valid values: Info, Debug, Warn, Error
+	// +optional
+	Level string `json:"level,omitempty"`
```

Comment:
- Since you're using `logrus.ParseLevel`, there are more valid values than these four. (Also they're lowercase afaics.)
- `omitempty` only works on pointer types.
```suggestion
	// Valid values: panic, fatal, error, warning, info, debug, trace
	// +optional
	Level *string `json:"level,omitempty"`
```